chart/newsfragments/49593.significant.rst

```diff
@@ -1,3 +1,3 @@
-Default Airflow image is updated to ``3.0.0``
+Default Airflow image is updated to ``3.0.2``
```

Comment:
Updated this newsfragment in-place since we haven't released chart with 3.0.0 anyways